sysdig/resource_sysdig_monitor_dashboard.go

```diff
@@ -53,6 +53,35 @@ func resourceSysdigMonitorDashboard() *schema.Resource {
 				ComputedWhen: []string{"public"},
 				Computed:     true,
 			},
+			"share": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"member": {
+							Type:     schema.TypeSet,
+							MaxItems: 1,
+							Required: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"type": {
+										Type:     schema.TypeString,
+										Required: true,
+									},
+									"id": {
+										Type:     schema.TypeInt,
+										Required: true,
```

Comment:
I guess we can't make this required if we also support `USER_TEAMS` based on the API request the Python client makes.